docs/components/eyebrow/index.html

```diff
@@ -0,0 +1,97 @@
+---
+title: Eyebrow
+minver: 0.24.0
+also:
+  components/eyebrow/test.html: Testing - Eyebrow
+  elements/hx-eyebrow-control: <hx-eyebrow-control>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    The {{page.title}} provides access to platform-level services and aggregate information
+     across a user's accounts at Rackspace.
+
+  </p>
+{% endblock %}
+
+{% block content %}
+
+<section>
+  <header>
+    <h2 id="icon-badge">Nav Menu</h2>
+  </header>
+
+  <div id="vue-eyebrowDemo">
+      <div class="eyebrow">
+          <a class="img-left brand-logo" href="#"> <img src="images/helix-logo.svg" alt="Logo" /></a>
+          <div class = "right-menu">
+            <a href="#"
+            :disabled="isDisabled"
+              @click="submitName()"
+            >
+              <hx-icon type="bell"></hx-icon>
+              <p>Notifications</p>
+            </a>
+
+            <a href="#">
+              <hx-icon type="ticketing"></hx-icon>
+              <p>Tickets</p>
+            </a>
+
+            <a href="#">
+              <hx-icon type="support"></hx-icon>
+              <p>Support</p>
+            </a>
+
+            <a href="#">
+              <hx-icon type="billing"></hx-icon>
+              <p>Billing</p>
+            </a>
+
+            <a href="#">
+              <hx-icon type="account"></hx-icon>
+              <p>Account</p>
+            </a>
+
+            <hx-disclosure class="eyebrow-menu" aria-controls="eyebrowMenuId" role="button" aria-expanded="true">
+              <span>User</span>
+              <hx-icon class="hxPrimary" type="angle-down" hx-defined=""></hx-icon>
+             </hx-disclosure>
+
+            <hx-menu id="eyebrowMenuId"  aria-hidden="false" position="bottom-start" role="menu" aria-expanded="true" open="">
```

Comment:
Please indent each attribute and place on a separate line.

Reply:
Changes have been done, Please review